splunk_add_on_ucc_framework/ucc_ui_lib/src/main/webapp/schema/schema.json

```diff
@@ -0,0 +1,1254 @@
+{
```

Comment:
I think, We should also add `schemaGenerator.py` along with `schema.json`  as `schema.json`  generated by it. So, If we want to change `schema.json`. We'll need it.

Reply:
Added